### PR TITLE
docs(node): add prioritize flags to node

### PR DESCRIPTION
**Pull request by nartc**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: docs/generated/packages/node/executors/node.json [hunk 1]
```diff
@@ -17,48 +17,56 @@
...
+        "default": "inspect",
+        "x-priority": "important"
       },
       "runtimeArgs": {
         "type": "array",
         "description": "Extra args passed to the node process.",
         "default": [],
-        "items": { "type": "string" }
+        "items": { "type": "string" },
+        "x-priority": "important"
```
Comment (jaysoo) [inline docs/generated/packages/node/executors/node.json:56]: I think `x-priority` shouldn't be added to most properties. You can leave the field off.

**Review by MaxKless — COMMENTED** [review]
I also feel like there's still too many important flags here. 
Please re-evaluate and make sure just those that really everyone will use a lot are marked as important. 

Also, should `standaloneConfig` be important when `workspace.json` is not the way we want to go in the future? Shouldn't it just default to `project.json`?

File: docs/generated/packages/node/generators/application.json [hunk 2]
```diff
@@ -18,8 +18,7 @@
       },
       "directory": {
```
Comment (MaxKless) [inline docs/generated/packages/node/generators/application.json:19]: as per the guidelines in asana, `directory` should be important

File: docs/generated/packages/node/generators/library.json [hunk 3]
```diff
@@ -51,31 +51,37 @@
...
         "enum": ["tsc", "swc"],
         "default": "tsc",
-        "description": "The compiler used by the build and test targets."
+        "description": "The compiler used by the build and test targets.",
+        "x-priority": "important"
       },
       "importPath": {
         "type": "string",
-        "description": "The library name used to import it, like `@myorg/my-awesome-lib`. Must be a valid npm name."
+        "description": "The library name used to import it, like `@myorg/my-awesome-lib`. Must be a valid npm name.",
```
Comment (MaxKless) [inline docs/generated/packages/node/generators/library.json:83]: Doesn't this have a strong, sensible default? Personally, I don't change the import path most of the time. Whats the reason for it being important here?

**Review by MaxKless — CHANGES_REQUESTED** [review]

File: docs/generated/packages/node/generators/library.json (hunk 3 above)
Comment (nartc) [reply docs/generated/packages/node/generators/library.json]: This is just me; I'm in the habit of setting `importPath` due to me creating publishable libraries all the time. In the context of `x-priority`, `important` can be taken off

**Review by MaxKless — APPROVED** [review]

**Pull request merged**